modules/op-installing-and-deploying-tekton-hub-on-an-openshift-cluster.adoc

```diff
@@ -19,7 +19,43 @@ If you are using Github Enterprise or Gitlab Enterprise, install and deploy {tek
 * Ensure that the {pipelines-title} Operator is installed in the default `openshift-pipelines` namespace on the cluster.
 
 [discrete]
-.Procedure
+.Procedure (Automatic)
```

Comment:
yeah, I agree with you
Can we have heading as 
* `Without Login and Rating` instead for automatic and `With Login and Rating` for manual
OR
* `Disable Login and Rating` instead of automatic and `Enable Login and Rating` for manual